### ISSUE TITLE
missing LICENSE

### ISSUE THREAD

**Issue by trombik** [iij/ipgen#61]
There is no LICENSE file. Would you provide one?

**Comment by ozaki-r** [comment]
Sure, I've added it on https://github.com/iij/ipgen/commit/27d6bba6a53a95203bc57d7fc2755eed3b8bfbfe.

**Comment by trombik** [comment]
Thanks. When a new release is available, the FreeBSD port will be ready, then.

**Comment by ozaki-r** [comment]
> Thanks. When a new release is available, the FreeBSD port will be ready, then.

OK, I see. We will probably make a release soon although there are a few new commits since the previous release because we won't have changes for the next few months.

**Comment by ozaki-r** [comment]
> > Thanks. When a new release is available, the FreeBSD port will be ready, then.
> 
> OK, I see. We will probably make a release soon although there are a few new commits since the previous release because we won't have changes for the next few months.

Oops. We used the term "release" as just "version", not "release" in GitHub terminology.
I've pushed a new version (v1.31). Is this enough for you? Or do you need a GitHub "release"?

**Comment by trombik** [comment]
No, the port doesn't need a release. A tag is enough for the port. Ports need a git ref that points to a single point of the git history, not a branch.

Thanks.

**Comment by ozaki-r** [comment]
I see. Thanks.
